src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Initializers/WebJobsTelemetryInitializer.cs

```diff
@@ -160,19 +179,24 @@ private bool TryApplyProperty(RequestTelemetry request, KeyValuePair<string, str
             {
                 request.Context.Operation.Name = property.Value;
                 request.Name = property.Value;
+
                 wasPropertySet = true;
             }
             else if (property.Key == LogConstants.SucceededKey &&
                 bool.TryParse(property.Value, out bool success))
             {
                 // no matter what App Insights says about the response, we always
-                // want to use the function's result for Succeeeded
+                // want to use the function's result for Succeeded
                 request.Success = success;
                 wasPropertySet = true;
 
                 // Remove the Succeeded property as it's duplicated
                 request.Properties.Remove(LogConstants.SucceededKey);
             }
+            else if (property.Key == LoggingConstants.ClientIpKey)
+            {
+                request.Context.Location.Ip = property.Value;
+            }
```

Comment:
Let's remove this ClientIpKey property once we set it., otherwise it's showing up in the `customDimensions`. Your GDPR comment makes me nervous to include the raw IP like that. So we should just set it on the `Context.Location.Ip`, and let App Insights do the right thing on the server (stripping it out). I do see the regional details being populated which is great.

Reply:
Good point, actually, this Activity.Tag and not the request property at the moment. I made sure it is not added into the request properties and added validation in the tests that we didn't add it